en/docs/reusable-components/choreo-connectors-list.md

```diff
@@ -0,0 +1,29 @@
+
+
+The following lists the connections you can use to interact with application-specific APIs:
+
+ - **Twilio**: Integrate with Twilio to perform operations such as sending messages, WhatsApp messages, and making voice calls.
+ - **Slack**: Integrate with Slack to perform operations such as sending a message to a channel, deleting a message, etc.
+ - **GitHub**: Integrate with GitHub to perform operations such as creating an issue, creating a pull request, etc. 
+ - **Gmail**: Integrate with Gmail to perform operations such as sending an email, receiving an email, etc.
+ - **Google Calendar**: Integrate with Google Calendar to perform operations such as creating an event, deleting an event, etc.
+ - **Google Sheets**: Integrate with Google Sheets to perform operations such as reading data, formatting data, etc.
+ - **Google Drive**: Integrate with Google Drive to perform operations such as creating files and folders, getting items saved in the drive (e.g., retrieving a file), renaming files, deleting files, etc. 
+ - **People API**: Integrate with the People API to manage your contacts. You can perform operations such as getting, listing, creating, deleting, and updating contacts.
+ - **Salesforce**: Integrate with Salesforce to perform operations such as obtaining organization information, creating a record, etc.
+ - **NetSuite**: Integrate with Netsuite to perform operations such as creating a record instance, creating a sublist, etc.
+ - **Azure EventHub**: Integrate with Azure to manage consumer groups, event hubs, events, publishers, and partitions. You can perform many operations for the mentioned items including getting, listing, creating, deleting, etc.
+ - **Azure CosmosDB**: Integrate with Azure CosmosDB to manage documents and stored procedures. You can perform operations for documents and stored procedures such as getting, creating, querying, replacing, deleting, etc.
+ - **MongoDB**: Integrate with MongoDB to manage databases, collections, and documents. You can perform operations such as getting databases, collections, and documents, inserting, updating, and deleting documents, closing the client, listing indices, etc. 
+ - **Redis**: Integrate with Redis to manage different data structures. You can perform a range of operations depending on the data structure.
+ - **AWS S3**: Integrate with AWS S3 to manage cloud storages. You can perform operations such as creating, listing and deleting buckets as well as objects.
+ - **AWS SQS**: Integrate with AWS SQS to manage messages and queues. You can perform operations such as creating queues, receiving messages, sending messages, deleting messages, and deleting queues.
+ - **COVID 19 API**: Integrate with the COVID 19 API to retrieve statistics relating to COVID-19. The GET operations you can perform include retrieving the status relating to the spread of COVID-19 and relating to the vaccine coverage per province, country, and continent. You can also retrieve global data. 
```

Comment:
should it be COVID 19 or COVID-19?

Reply:
related to?

Reply:
Shall we rephrase this to say " You can perform operations such as retrieving the statistics regarding the spread of the virus in USA and globally. You can also retrieve statistics relating to the vaccine coverage per province, country, and continent."

Reply:
The name of the API in "COVID 19" without the dash, but in general, it is written with the dash.

Reply:
Created an issue a couple of days back to get Covid 19 changed to COVID-19 [1]. This task is in progress at the moment.
[1] https://github.com/wso2-enterprise/choreo/issues/5680